frontend/test/metabase/entities/schemas.unit.spec.js

```diff
@@ -227,6 +227,7 @@ describe("schema entity", () => {
       const nextState = Schemas.reducer(
         {
           [ROOT_COLLECTION_VIRTUAL_SCHEMA]: {
+            id: ROOT_COLLECTION_VIRTUAL_SCHEMA,
```

Comment:
Fix the test, because the change in this PR expects the data to contain an `id` property.